doc/geany.txt

```diff
@@ -3099,37 +3096,39 @@ is shown in the following table.
 |              |                     | ~/.config/geany/filedefs |                   |                               |
 +--------------+---------------------+--------------------------+-------------------+-------------------------------+
 
-The following notes on the table reference cells by coordinate as (group,source):
+The following notes on the table may reference cells by coordinate as *(group, source)*:
 
-* General - for filetypes.xxx substitute the appropriate extension for
+* Filetype filenames - for filetypes.xxx substitute the appropriate extension for
   the filetype of the current document for xxx - see `filenames`_.
 
 * System Filetypes - Labels loaded from these sources are locale sensitive
   and can contain translations.
 
-* (Filetype, Project File) and (Filetype, Preferences) - preferences use a full
+* *(Filetype build, Project and Preferences)* - preferences use a full
   filetype file so that users can configure all other filetype preferences
   as well.  Projects can only configure menu items per filetype.  Saving
   in the project file means that there is only one file per project not
   a whole directory.
 
-* (Filetype-Independent, System Filetype) - although conceptually strange, defining
+* *(Filetype-Independent build, System Filetype)* - although conceptually strange, defining
   filetype-independent commands in a filetype file, this provides the ability to
   define filetype dependent default menu items.
 
-* (Execute, Project File) and (Execute, Preferences) - the project independent
+* *(Execute, Project and Preferences)* - the project independent
   execute and preferences independent execute commands can only be set by hand
   editing the appropriate file, see `Preferences file format`_ and `Project file
   format`_.
 
-Build menu commands dialog
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Most of the configuration of the build menu is done through the Build Menu
-Commands Dialog.  You edit the configuration sourced from preferences in the
-dialog opened from the Build->Build Menu Commands item and you edit the
-configuration from the project in the build tab of the project preferences
-dialog.  Both use the same form shown below.
+Set Build Commands dialog
+^^^^^^^^^^^^^^^^^^^^^^^^^
+ 
+Most of the configuration of the build menu is done through the *Set 
+Build Commands* dialog. When no project is open, you can edit the 
```

Comment:
Could use `` `Set Build Commands dialog`_ `` or whatever is the hyperlink syntax.